polyfills/UserTiming/detect.js

```diff
@@ -0,0 +1 @@
+'performance' in window && typeof window.performance.getEntriesByType === 'function' && typeof window.performance.mark === 'function'

```

Comment:
Prefer `this` to `window`, since we do not guarantee that `window` will not have been overridden.